helmfile/charts/gatekeeper-metrics/templates/gatekeeper-audit-metrics.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: gatekeeper-metrics
+    chart: gatekeeper-metrics
+    control-plane: audit-controller
+    type: metrics
+  name: gatekeeper-audit-metrics
```

Comment:
Should we use something like this https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/charts/harbor-backup/templates/harbor-backup-job.yaml#L4 to make it more resilient of multiple releases.

Reply:
Sure I can change it to that